uni/lib/Makefile

```diff
@@ -4,13 +4,14 @@ include $(BASE)/Makedefs.uni
 .PHONY: all clean deps
 
 UFILES=gui.u file_dlg.u db.u \
- address.u base64handler.u basicclasscoding.u class.u classcoding.u \
+ address.u args.u base64handler.u basicclasscoding.u blockread.u class.u classcoding.u \
  cltable.u comparator.u complex.u compoundedit.u contentdisposition.u contenttype.u \
- cvsuser.u cvsutil.u decode.u encode.u encodinghandler.u error.u format.u \
+ cvsuser.u cvsutil.u decode.u encode.u encodinghandler.u error.u exception.u \
+ fcn_util.u format.u \
  group.u httpclient.u json.u langprocs.u listener.u \
- mailbox.u netclient.u mailmisc.u message.u messagehandler.u method.u \
- money.u msg.u multipart.u multiparthandler.u noophandler.u object.u \
- popclient.u predicat.u process.u pushback.u qsort.u quotedprintablehandler.u \
+ mailbox.u mailmisc.u mapbytes.u message.u messagehandler.u method.u \
+ money.u msg.u multipart.u multiparthandler.u netclient.u noophandler.u object.u \
+ popclient.u predicat.u process.u property.u pushback.u qsort.u quotedprintablehandler.u \
```

Comment:
Not sure if any of the remaining files would have the same or similar dependency issue.  I suggest trying to build locally first and fix any issue if there are any.